.infrastructure/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-todoapp
+  namespace: todoapp
+spec:
+  storageClassName: standart
```

Comment:
The storage class name 'standart' seems to be a typo. It should likely be 'standard' unless 'standart' is a custom-defined storage class in your Kubernetes cluster.